tt_metal/impl/device/device.hpp

```diff
@@ -116,9 +116,10 @@ class Device {
     CoreCoord dram_grid_size() const;
 
     CoreType core_type_from_virtual_core(const CoreCoord& virtual_coord) const;
-
+    // Given a Virtual coordinate in noc_index space, get the equivalent coordinate in Virtual NOC0 space
     CoreCoord virtual_noc_coordinate(uint8_t noc_index, CoreCoord coord) const;
-
+    // Given a coordinate in Virtual NOC0 Space, get the equivalent coordinate in Virtual noc_index space
```

Comment:
think the comments here are swapped?

Reply:
These names are confusing, I'm sorry.
The convention here is: 
`virtual_noc0_coordinate` accepts coordinates in NOC0 space and will return Virtual NOC coords in noc_index space
`virtual_noc_coordinate` accepts coordinates in noc_index space and will return Virtual NOC0 coordinates.

The index in the name refers to the the input coordinate system. Let me think of a better naming scheme once BH CI is unblocked.